src/services/telementryService.js

```diff
@@ -56,43 +56,54 @@ export const initialize = ({ context, config, metadata }) => {
 };
 
 export const start = duration => {
-  CsTelemetryModule.instance.telemetryService.raiseStartTelemetry({
-    options: getEventOptions(),
-    edata: {
-      type: 'content',
-      mode: 'play',
-      stageid: url,
-      duration: Number((duration / 1e3).toFixed(2)),
-    },
-  });
+  if (process.env.REACT_APP_TELEMETRY_MODE === 'ET' || process.env.REACT_APP_TELEMETRY_MODE === 'NT' || process.env.REACT_APP_TELEMETRY_MODE === 'DT') {
+    CsTelemetryModule.instance.telemetryService.raiseStartTelemetry({
+      options: getEventOptions(),
+      edata: {
+        type: 'content',
+        mode: 'play',
+        stageid: url,
+        duration: Number((duration / 1e3).toFixed(2)),
+      },
+    });
+  }
+
 };
 
 export const response = (context, options) => {
-  CsTelemetryModule.instance.telemetryService.raiseResponseTelemetry(
-    {
-      ...context,
-    },
-    getEventOptions()
-  );
+  if (process.env.REACT_APP_TELEMETRY_MODE === 'ET' || process.env.REACT_APP_TELEMETRY_MODE === 'NT' || process.env.REACT_APP_TELEMETRY_MODE === 'DT') {
```

Comment:
need to update the login to => 
 (this.TELEMETRY_MODE === 'ET' && currentMode === 'ET') ||
      (this.TELEMETRY_MODE === 'NT' &&
        (currentMode === 'ET' || currentMode === 'NT')) ||
      (this.TELEMETRY_MODE === 'DT' &&
        (currentMode === 'ET' || currentMode === 'NT' || currentMode === 'DT'))
    )